x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/alerts/user_assignment.md

```diff
@@ -0,0 +1,186 @@
+# Alert User Assignment
+
+This is a test plan for the Alert User Assignment feature
+
+Status: `in progress`. The current test plan covers functionality described in [Alert User Assignment](https://github.com/elastic/security-team/issues/2504) epic.
+
+## Useful information
+
+### Tickets
+
+- [Alert User Assignment](https://github.com/elastic/security-team/issues/2504) epic
+- [Add test coverage for Alert User Assignment](https://github.com/elastic/kibana/issues/171307)
+- [Write a test plan for Alert User Assignment](https://github.com/elastic/kibana/issues/171306)
+
+### Terminology
+
+- **Assignee**: The user assigned to an alert.
+
+- **Assignees field**: The alert's `kibana.alert.workflow_assignee_ids` field which contains an array of assignees IDs.
+
+- **Assignee's avatar**: The avatar of an assignee. Can be either user profile picture if uploaded by the user or initials of the user.
+
+- **Assignees count badge**: The badge with the number of assignees.
+
+### Assumptions
+
+- Assignees are stored as an array of users IDs in alert's `kibana.alert.workflow_assignee_ids` field
+- The feature is available under the Basic license
+- There are multiple (five or more) available users which could be assigned to alerts
+- User need to have editor or higher privileges to assign users to alerts
+- Mixed states are not supported by the current version of User Profiles component
+
```

Comment:
Another assumption is that when it says `displayed in UI` in the scenarios below, it is referring to the alerts table and the alert details flyout.

Reply:
done